app/Services/Account/Settings/ExportAccount.php

```diff
@@ -0,0 +1,1366 @@
+        $this->exportContact($data);
+        $this->exportActivityContact($data);
+        $this->exportActivityStatistic($data);
+        $this->exportActivityTypeCategory($data);
+        $this->exportActivityType($data);
+        $this->exportAddress($data);
+        $this->exportCall($data);
+        $this->exportCompany($data);
+        $this->exportContactFieldType($data);
+        $this->exportContactField($data);
+        $this->exportContactTag($data);
+        $this->exportContact($data);
+        $this->exportConversation($data);
+        $this->exportDays($data);
+        $this->exportDebt($data);
+        $this->exportDocument($data);
+        $this->exportEmotionCall($data);
+        $this->exportEntries($data);
+        $this->exportGender($data);
+        $this->exportGift($data);
+        $this->exportInvitation($data);
+        $this->exportJournalEntry($data);
+        $this->exportLifeEventCategory($data);
+        $this->exportLifeEventType($data);
+        $this->exportLifeEvent($data);
+        $this->exportMessage($data);
+        $this->exportMetaDataLoveRelationship($data);
+        $this->exportModule($data);
+        $this->exportNote($data);
+        $this->exportNotification($data);
+        $this->exportOccupation($data);
+        $this->exportPet($data);
+        $this->exportPhoto($data);
+        $this->exportPlace($data);
+        $this->exportRecoveryCode($data);
+        $this->exportRelationTypeGroup($data);
+        $this->exportRelationType($data);
+        $this->exportRelationship($data);
+        $this->exportReminderOutbox($data);
+        $this->exportReminderRule($data);
+        $this->exportReminderSent($data);
+        $this->exportReminder($data);
+        $this->exportSpecialDate($data);
+        $this->exportTag($data);
+        $this->exportTask($data);
+        $this->exportTermUser($data);
+        $this->exportUser($data);
+        $this->exportWeather($data);
+        $this->exportContactPhoto($data);
+
+        $this->sql = $this->sql.'SET FOREIGN_KEY_CHECKS=1;';
+
+        return $this->sql;
+    }
+
+    private function buildInsertSQLQuery(string $tableName, string $foreignKey, array $columns, array $data)
+    {
+        $accountData = DB::table($tableName)
+            ->select($columns)
+            ->where($foreignKey, $data['account_id'])
+            ->get();
+
+        if (! $accountData) {
+            throw new NoAccountException();
+        }
+
+        if ($accountData->count() == 0) {
+            return;
+        }
+
+        // adding a ` for each column
+        $listOfColumns = $columns;
+        foreach ($listOfColumns as $key => $value) {
+            $listOfColumns[$key] = '`'.$value.'`';
+        }
+        $listOfColumns = implode(',', $listOfColumns);
+
+        foreach ($accountData as $singleSQLData) {
+            $columnValues = [];
+
+            $this->sql = $this->sql.'INSERT IGNORE INTO '.$tableName.' ('.$listOfColumns.') values (';
```

Comment:
We may (?) add the prefix table in here.

Reply:
ohhh right yes you are right.